<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="frameRangeOptions_groupBox">
     <property name="title">
      <string>Frames</string>
     </property>
     <layout class="QFormLayout" name="options1_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="frameRangeModeLabel">
        <property name="text">
         <string>Frame Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="frameRangeModeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="frameRangeLabel">
        <property name="text">
         <string>Start/End</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="frameRangeMode_layout">
        <item>
         <widget class="QSpinBox" name="frameRangeStartSpinBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="frameRangeEndSpinBox">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="options_groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QFormLayout" name="options2_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="bundleUnlockRelockLabel">
        <property name="text">
         <string>Automatic Bundle Unlock/Relock</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="bundleUnlockRelockCheckBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
